python-avd/pyavd/_eos_designs/structured_config/metadata/cv_pathfinder.py

```diff
@@ -204,11 +204,10 @@ def _metadata_vrfs(self: AvdStructuredConfigMetadata) -> list:
         return strip_empties_from_list(metadata_vrfs)
 
     def _get_vni_for_vrf_name(self: AvdStructuredConfigMetadata, vrf_name: str) -> int:
-        if vrf_name not in self.inputs.wan_virtual_topologies.vrfs or (wan_vni := self.inputs.wan_virtual_topologies.vrfs[vrf_name].wan_vni) is None:
-            if vrf_name == "default":
-                return 1
+        if vrf_name in self.inputs.wan_virtual_topologies.vrfs and (wan_vni := self.inputs.wan_virtual_topologies.vrfs[vrf_name].wan_vni) is not None:
```

Comment:
You added a helper in shared_utils for almost the same as this. Maybe the helper would be more useful by returning the wan_vni or None if not found.